js/renderFunctions.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+    Text,
+    TouchableHighlight,
+    View,
+  } from 'react-native';
+import { Image } from 'react-native-elements'
+export const renderTouchables = (key, imageSource, p, styles) => {
```

Comment:
What is "p"? Can we use a more descriptive param?

Reply:
changed to item